src/OSPSuite.Infrastructure.Import/Services/IDataImporter.cs

```diff
@@ -56,5 +56,13 @@ DataImporterSettings dataImporterSettings
       /// </summary>
       /// <returns>a list of meta data categories</returns>
       IList<MetaDataCategory> DefaultMetaDataCategories();
+
+      /// <summary>
+      /// Compares if two data repositories come from the same data
+      /// </summary>
+      /// <param name="sourceDataRepository"></param>
+      /// <param name="targetDataRepository"></param>
+      /// <returns></returns>
+      bool AreFromSameSource(DataRepository sourceDataRepository, DataRepository targetDataRepository);
```

Comment:
Isn't the name a bit misleading since it in fact SKIPS the file name (the source)

Reply:
and the code comment says "Ignore source" while the variable is called FILE lol. I think we should make all of that consistent